src/features/swap/index.tsx

```diff
@@ -0,0 +1,207 @@
+    amount: string
+  }
+}
+
+const appData: SafeAppData = {
+  id: 1,
+  url: 'https://app.safe.global',
+  name: 'Safe Swap',
+  iconUrl: 'https://app.safe.global/icon.png',
+  description: 'Safe Apps',
+  chainIds: ['1', '100'],
+  accessControl: { type: SafeAppAccessPolicyTypes.NoRestrictions },
+  tags: ['safe-apps'],
+  features: [SafeAppFeatures.BATCHED_TRANSACTIONS],
+  socialProfiles: [],
+}
+const SwapWidget = ({ sell }: Params) => {
+  const chainId = useChainId()
+  const { palette } = useTheme()
+  const darkMode = useDarkMode()
+  const dispatch = useAppDispatch()
+
+  const [toasts, setToasts] = useState<String[]>([])
+
+  const groupKey = 'swap-order-status'
+  const listeners = useMemo<CowEventListeners>(() => {
+    return [
+      {
+        event: CowEvents.ON_TOAST_MESSAGE,
+        handler: (event) => {
+          console.info('🍞 New toast message:', event)
+          const { message, messageType, data } = event
+          switch (messageType) {
+            // @ts-ignore
+            case 'ORDER_CREATED':
+              dispatch(
+                showNotification({
+                  title: 'Order created',
+                  message: 'Order created but waiting for presignature',
+                  groupKey,
+                  variant: 'info',
+                }),
+              )
+              break
+            // @ts-ignore
+            case 'ORDER_PRESIGNED':
+              dispatch(
+                showNotification({
+                  title: 'Order presigned waiting for fullfillment',
+                  message: 'Order was presigned and waiting for match',
+                  groupKey,
+                  variant: 'info',
+                }),
+              )
+              break
+            // @ts-ignore
+            case 'ORDER_FULFILLED':
+              dispatch(
+                showNotification({
+                  title: 'Order fullfilled',
+                  message: 'Order was fullfilled',
+                  groupKey,
+                  variant: 'info',
+                }),
+              )
+              break
+            // @ts-ignore
+            case 'ORDER_EXPIRED':
+              dispatch(
+                showNotification({
+                  title: 'Order expired',
+                  message,
+                  groupKey,
+                  variant: 'warning',
+                }),
+              )
+              break
+            // @ts-ignore
+            case 'ORDER_CANCELLED':
+              dispatch(
+                showNotification({
+                  title: 'Order cancelled',
+                  message,
+                  groupKey,
+                  variant: 'warning',
+                }),
+              )
+              break
+          }
+        },
+      },
+    ]
+  }, [dispatch])
+
+  const [params, setParams] = useState<CowSwapWidgetParams | null>(null)
+  useEffect(() => {
+    setParams({
+      appCode: 'Safe Wallet Swaps', // Name of your app (max 50 characters)
```

Comment:
nit: I would add here the name of the app: `Safe Wallet Web` or maybe app.safe.global if you prefer